frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -276,13 +287,16 @@ export const createDeployment = (
     },
   };
 
-  return k8sCreate(DeploymentModel, deployment, dryRun ? dryRunOpt : {});
+  return oldDeployment
+    ? k8sUpdate(DeploymentModel, deployment, namespace, name)
```

Comment:
No need to pass namespace and name explicitly.
```suggestion
    ? k8sUpdate(DeploymentModel, deployment)
```